transmart-data/updatedb/release-17.1-HYVE-4/postgres/migrate.sql

```diff
@@ -3,4 +3,5 @@
 \i ../../../ddl/postgres/biomart_user/views/patient_num_boundaries.sql
 \i ../../../ddl/postgres/biomart_user/materialized_views/study_concept_bitset.sql
 \i ../../../ddl/postgres/biomart_user/views/patient_set_bitset.sql
-\i ../../../ddl/postgres/biomart_user/views/study_concept_patient_set_bitset.sql
\ No newline at end of file
+\i ../../../ddl/postgres/biomart_user/views/study_concept_patient_set_bitset.sql
+\i fix_neg_patient_nums.sql
```

Comment:
Please exclude from the migrate script, but do document the script.
This does not have to be applied as mandatory migration step for production instances.